include/i2c.h

```diff
@@ -55,6 +56,24 @@ extern "C" {
 /** Controller to act as Master. */
 #define I2C_MODE_MASTER			(1 << 4)
 
+static inline int __i2c_map_dt_bitrate(u32_t bitrate)
```

Comment:
does this belong in the i2c API header? why not have this in dt-bindings where other defines used by the function also exist?